index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en-US">
   <head>
     <meta charset="utf-8">
-    <meta http-equiv="x-ua-compatible" content="IE=Edge">
+    <meta http-equiv="Content-Security-Policy" content="default-src 'none'; style-src 'self' 'unsafe-inline' *.princeton.edu; script-src 'self' 'unsafe-eval' *.princeton.edu; connect-src 'self' https://api.honeybadger.io https://docs.google.com *.googleusercontent.com; font-src 'self'; base-uri 'none'; img-src 'self' *.princeton.edu data:;">
```

Comment:
Nice!

Reply:
It was blocking the test error to setup Honeybadger, so it was needed to get everything working again.